examples/custodian.ttl

```diff
@@ -78,13 +78,17 @@ sdc:SignatureShape a sh:NodeShape ;
                 sdc:identifier,
                 sdc:hasSignatureShape.
 
-###################################################################################################################
+# ###################################################################################################################

```

Comment:
I don't know if we care about this, but there's a space that made its way here. If it was made by mistake we can remove it now before we forget about it.

```suggestion
####################################################################################################################

```

Reply:
This is just a bunch of hashtags for me to section the ontology in different parts. it has no functional meaning, and will not be affected by a space :)

very sharp though ahaha